mingw-w64.rb

```diff
@@ -0,0 +1,197 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows with GCC."
+  homepage "https://mingw-w64.org/"
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v4.0.6.tar.bz2"
+  sha256 "0c407394b0d8635553f4fbca674cdfe446aac223e90b4010603d863e4bdd015c"
+
+  depends_on "gcc6" => :build
+  depends_on "texinfo" => :build
+  depends_on "gmp"
+  depends_on "mpfr"
+  depends_on "libmpc"
+  depends_on "cloog"
+  depends_on "isl014"
+
+  resource "binutils" do
+    url "https://ftpmirror.gnu.org/binutils/binutils-2.26.1.tar.bz2"
+    sha256 "39c346c87aa4fb14b2f786560aec1d29411b6ec34dce3fe7309fe3dd56949fd8"
+  end
+
+  resource "gcc" do
+    url "https://ftpmirror.gnu.org/gcc/gcc-6.2.0/gcc-6.2.0.tar.bz2"
+    sha256 "9944589fc722d3e66308c0ce5257788ebd7872982a718aa2516123940671b7c5"
+  end
+
+  def install
+    target_archs = ["i686-w64-mingw32", "x86_64-w64-mingw32"]
+    target_archs.each do |target_arch|
+      resource("binutils").stage do
+        install_prefix = "#{libexec}/#{target_arch}"
+        args = %W[
+          CC=gcc-6
+          CXX=g++-6
+          CPP=cpp-6
+          LD=gcc-6
+          --target=#{target_arch}
+          --disable-werror
+          --disable-multilib
+          --prefix=#{libexec}
+        ]
+
+        system "./configure", *args
+        system "make"
+        system "make", "install-strip"
+
+        chdir install_prefix.to_s do
+          ln_s install_prefix.to_s, "#{libexec}/binutils"
+        end
+      end
+
+      ENV.prepend_path "PATH", "#{libexec}/bin"
+
+      install_prefix = "#{libexec}/#{target_arch}"
+      args = %W[
+        --host=#{target_arch}
+        --prefix=#{install_prefix}
+        --enable-sdk=all
+        --enable-secure-api
+      ]
+
+      mkdir "build-headers" do
+        system "../mingw-w64-headers/configure", *args
+        system "make"
+        system "make", "install"
+      end
+
+      # TODO sgroat
+      #chdir install_prefix.to_s do
+      #  ln_s install_prefix.to_s, "#{libexec}/mingw"
+      #end
+
```

Comment:
Not OK.
This symbolic link is used in mingw compilation phase.

ref: https://sourceforge.net/p/mingw-w64/wiki2/Cross%20Win32%20and%20Win64%20compiler/